Cargo.toml

```diff
@@ -2,12 +2,29 @@
 resolver = "2"
 members = [
     "crates/ibc",
+    "crates/ibc-data-types",
+    "crates/ibc-primitives",
+    "crates/ibc-core",
     "crates/ibc-apps",
-    "crates/ibc-apps/ics20-transfer",
-    "crates/ibc-apps/ics20-transfer/types",
     "crates/ibc-derive",
     "crates/ibc-testkit",
     "crates/ibc-query",
+    "crates/ibc-core/ics02-client",
+    "crates/ibc-core/ics02-client/types",
+    "crates/ibc-core/ics03-connection",
+    "crates/ibc-core/ics03-connection/types",
+    "crates/ibc-core/ics04-channel",
+    "crates/ibc-core/ics04-channel/types",
+    "crates/ibc-core/ics23-commitment/types",
+    "crates/ibc-core/ics24-host",
+    "crates/ibc-core/ics24-host/types",
+    "crates/ibc-core/ics24-host/tendermint",
+    "crates/ibc-core/ics25-handler",
+    "crates/ibc-core/ics25-handler/types",
+    "crates/ibc-core/ics26-routing",
+    "crates/ibc-core/ics26-routing/types",
```

Comment:
btw, cargo workspace members [can take glob patterns](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-members-and-exclude-fields). But it's up to you if you want to use that here.

Reply:
Cool, I’ll apply this once we move all the crates from the `crates/..` dir into the root of the repo.

Reply:
Second thoughts, I'd rather keep things the same, since it's easier for development to selectively comment out a crate and see how it behaves.

Reply:
Yea fine with me :slightly_smiling_face: just wanted to make sure that you are aware of the option.